### ISSUE TITLE
Arcolinux unable to login, sending back to login screen

### ISSUE THREAD

**Issue by a-sebi** [WillPower3309/awesome-dotfiles#41]
I'm running Arcolinux and have installed all dependencies (uninstalled awesomewm from Arcolinux) and after copying the awesome config folder into my .config folder, I'm unable to login. I login but it sends me back to the login screen. Note that I managed to get your config working when replacing the awesome folder while logged in and restarting awesome. 

Once I logged out, I can't get back in. Sorry, tried to contact directly as this is obviously something on my end but any help is appreciated (new to awesomewm)! All dependencies are installed, just wondering what would cause me to get sent back to login screen.

Tested this on a different distro (Archcraft) and it seemed to work fine though.

**Comment by WillPower3309** [comment]
Does awesomewm work on arco without my config applied?

**Comment by a-sebi** [comment]
Yes it works fine without your config, switching to themes like powerarrow-dark also work without issue. I tried uninstalling awesomewm completely, along with all of the dependencies packaged with Arcolinux's version and just installing awesomwm by itself but the issue was still there.

I've tried googling but there doesn't seem to be much about logging in and facing a blank screen, then getting put back into login screen. Logs from journalctl or lightdm don't seem to show any errors. The only relevant thing I can find in the lightdm logs is below. Should I be checking any other log files?

[+284.60s] DEBUG: Session pid=2902: Running command /etc/lightdm/Xsession awesome
[+284.60s] DEBUG: Creating shared data directory /var/lib/lightdm-data/asebi
[+284.60s] DEBUG: Session pid=2902: Logging to .xsession-errors
[+284.62s] DEBUG: Activating VT 7
[+284.62s] DEBUG: Activating login1 session 5
[+284.62s] DEBUG: Seat seat0 changes active session to
[+284.62s] DEBUG: Seat seat0 changes active session to 5
[+284.62s] DEBUG: Session 5 is already active
[+285.62s] DEBUG: Session pid=2902: Exited with return value 1
[+285.62s] DEBUG: Seat seat0: Session stopped
[+285.62s] DEBUG: Seat seat0: Stopping display server, no sessions require it
[+285.62s] DEBUG: Sending signal 15 to process 2821
[+285.83s] DEBUG: Process 2821 terminated with signal 6
[+285.83s] DEBUG: XServer 0: X server stopped

**Comment by WillPower3309** [comment]
what version of awesomewm are you running?

**Comment by a-sebi** [comment]
Version 4.3-2.

**Comment by a-sebi** [comment]
Found some info in `.xsession-errors` that may be relevant?

```
2021-06-12 11:48:25 W: awesome: wibox.layout.margin has been renamed to wibox.container.margin.
stack traceback:
	/usr/share/awesome/lib/gears/debug.lua:95: in function 'gears.debug.deprecate'
	/usr/share/awesome/lib/gears/debug.lua:131: in field 'margin'
	/home/asebi/.config/awesome/components/pastel/top-panel.lua:47: in function 'components.pastel.top-panel.create'
	/home/asebi/.config/awesome/pastel.lua:53: in local 'func'
	/usr/share/awesome/lib/awful/screen.lua:438: in function 'awful.screen.connect_for_each_screen'
	/home/asebi/.config/awesome/pastel.lua:35: in function 'pastel.initialize'
	/home/asebi/.config/awesome/rc.lua:79: in main chunk
2021-06-12 11:48:25 W: awesome: wibox.layout.margin has been renamed to wibox.container.margin.
stack traceback:
	/usr/share/awesome/lib/gears/debug.lua:95: in function 'gears.debug.deprecate'
	/usr/share/awesome/lib/gears/debug.lua:131: in field 'margin'
	/home/asebi/.config/awesome/components/pastel/top-panel.lua:51: in function 'components.pastel.top-panel.create'
	/home/asebi/.config/awesome/pastel.lua:53: in local 'func'
	/usr/share/awesome/lib/awful/screen.lua:438: in function 'awful.screen.connect_for_each_screen'
	/home/asebi/.config/awesome/pastel.lua:35: in function 'pastel.initialize'
	/home/asebi/.config/awesome/rc.lua:79: in main chunk
XIO:  fatal IO error 0 (Success) on X server ":0"
      after 20 requests (20 known processed) with 0 events remaining.
[ 12/06/21 11:48:26.026 x_fetch_region ERROR ] Failed to fetch rectangles (No error)
X connection to :0 broken (explicit kill or server shutdown).
2021-06-12 11:48:26 E: awesome: client_ignore_enterleave_events:1231: X server connection broke (error 1)
```

**Comment by WillPower3309** [comment]
Okay I think I see the issue, I'll investigate and hopefully fix for tomorrow

**Comment by sheeepdev** [comment]
When will this be solved :(

**Comment by WillPower3309** [comment]
My apologies, this slipped away as I was looking into other things. I'll get back to this one shortly

**Comment by sheeepdev** [comment]
I was having this same problem on EndeavourOS, and I'm trying on manjaro now, hopefully, by changing distros it may be solved, ill update you

**Comment by sheeepdev** [comment]
UPDATE:

Works on manjaro

**Comment by WillPower3309** [comment]
> UPDATE:
> 
> Works on manjaro

Thanks for the info! I suspect that some distros package a newer version of awesomewm. I'm going to try to take a look at everything to see if I can update without breaking other distro versions
